video/out/wayland_common.c

```diff
@@ -2097,6 +2103,18 @@ static bool create_input(struct vo_wayland_state *wl)
     return 0;
 }
 
+static int xdg_activate(struct vo_wayland_state *wl)
+{
+    if (wl->xdg_activation == NULL)
+        return 1;
```

Comment:
```suggestion
    if (!wl->xdg_activation)
        return;
```

Minor style thing but ditto for below. Also if the return value isn't being used, we should just void here. And don't I foresee any reason to log a failure case here.